src/screens/SendSheet.js

```diff
@@ -482,7 +484,7 @@ export default function SendSheet(props) {
       from: accountAddress,
       gasLimit: gasLimitToUse,
       gasPrice: selectedGasPrice.value?.amount,
-      nonce: null,
+      nonce: await getNextNonce(),
```

Comment:
say the `getNextNonce` blows up on us or hangs, at which point we may want to cut our losses and at least be able to send the txn with `nonce: null` vs not being able to send at all @brunobar79  thoughts?

Reply:
this applies for the other places as well

Reply:
also i dunno what the timeout is on `getNextNonce` but it should be closer to 5-10seconds

Reply:
Now catching exceptions in `getNextNonce` and returning null. Per Bruno's Slack comments, not doing anything regarding timeouts here.